### ISSUE TITLE
Documentation: Contributing TOC links to missing sections

### ISSUE THREAD

**Issue by hilary** [exercism/x-api#152]
The TOC for the [CONTRIBUTING](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md) Guide contains entries for

* config.json
* configlet
* Problem API
* Track Anatomy

none of which is in the Guide. While the Track Anatomy entry links to its new location in x-common, the other three links are ghost links.

Should those links be removed, or should sections on those topics be added? I'd be happy to submit a PR removing them (or marking them as 'coming soon' or whatever). As it is, it's confusing to noobs like me :)

**Comment by Insti** [comment]
`config.json` should be documented somewhere else.  - Delete
`configlet` I think lives somewhere else now - Probably Delete
`problem API` sounds like something that should be here. - Keep but check it is still relevant.
`Track Anatomy` should be documented somewhere else - Delete

**Comment by kytrinyx** [comment]
I think if the _problem API_ is a dead link, go ahead and delete it. If we find that we need detailed documentation about x-api, it should live in the x-api repo, and if we're missing high-level documentation about x-api, it should probably live in the architecture overview in the docs repo.

**Comment by Radix16** [comment]
I recently implemented an exercise in the Javascript track.   The toughest part of implementing the exercise was tracking down the information to complete the config.json file.   I found the repository for the configlet tool.  However, I was unable to find any comprehensive documentation for what belonged in the config.json.   I agree with @hilary.   These topics should contain entries or at least links to the real documentation.   Deleting them will only discourage people from contributing.
